daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2126,9 +2157,11 @@ func (devices *DeviceSet) removeDevice(devname string) error {
 	return err
 }
 
-func (devices *DeviceSet) cancelDeferredRemoval(info *devInfo) error {
+func (devices *DeviceSet) cancelDeferredRemoval(info *devInfo) (bool, error) {
```

Comment:
And this function will probably return ErrEnxio while it is retrying in busy loop.